app/services/cloud_topology_service.rb

```diff
@@ -9,6 +9,8 @@ class CloudTopologyService < TopologyService
     :cloud_tenants      => [:tags, :vms => :tags],
   ]
 
+  @kinds = %i(CloudManager AvailabilityZone CloudTenant Vm Tag)
```

Comment:
I'd like to keep it consistent with the other instance two variables above, but will change it later.